sql/core/src/test/resources/sql-tests/inputs/interval.sql

```diff
@@ -88,3 +88,46 @@ select justify_interval(interval '1 month -59 day 25 hour');
 select justify_days(interval '1 month 59 day -25 hour');
 select justify_hours(interval '1 month 59 day -25 hour');
 select justify_interval(interval '1 month 59 day -25 hour');
+
+-- interval output style
```

Comment:
we can create a `interval-display.sql` test file to put these test cases, and then a `interval-display-sql-standard.sql` which does
```
--SET spark.sql.intervalOutputStyle = SQL_STANDARD;
--import interval-display.sql
```

also a `interval-display-iso_8601.sql`

Reply:
```
--SET spark.sql.intervalOutputStyle = SQL_STANDARD
--import interval-display.sql
```
 these 2 don't fit each other well,
When regenerating golden files, the set operations will not happen, considering this as a bug,
also cc @maropu 

https://github.com/apache/spark/blob/40ea4a11d7f1534023669f0b81faf5d398174e46/sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala#L288-L295

Reply:
https://github.com/apache/spark/pull/26557, In these PR, the golden files are generated with this fix, because it has nothing to do with this PR, so I just raised a follow-up to the ticket SPARK-29873 which involved this feature.